index.html

```diff
@@ -29,30 +29,344 @@ <h1>
+                            <a href=""><span class="language">Python</span></a>
+                        </li>
+                    </ul>
+                    <ul class="knowledge">
+                        <li>
+                            <h3 class="h3-about-me-outer">
+                                <span class="floated-bullet bullet2"></span>
+                                <span class="h3-about-me">
+                                    Frameworks
+                                </span>
+                            </h3>
+                        </li>
+                        <li>
+                            <a href=""><span class="language">Bootstrap</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">Ruby on Rails</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">RSPec</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">CapyBara</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">Selenium</span></a>
+                        </li>
+                    </ul>
+                    <ul class="knowledge">
+                        <li>
+                            <h3 class="h3-about-me-outer">
+                                <span class="floated-bullet bullet3"></span>
+                                <span class="h3-about-me">
+                                    Skills
+                                </span>
+                            </h3>
+                        </li>
+                        <li>
+                            <a href=""><span class="language">Codekit</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">Github</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">Codepen</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">Gitlab</span></a>
+                        </li>
+                        <li class="language">
+                            <a href=""><span class="language">Terminal</span></a>
+                        </li>
+                    </ul>
+                </div>
+            </section>
+        </section>
+        <section class="form-container">
+            <h3 class="form-text">
+                I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.
+            </h3>
+            <form action="#" method="post" class="form">
```

Comment:
- [ ] [REQUIRED] I kindly suggest that you use your form action to integrate with [formspree](https://formspree.io/html/) service as this will make it possible for you to receive user submissions to your email address.

Example:
`<form action="https://formspree.io/f/{form_id}" method="post" class="form">`
            
`</form>`